modules/op-release-notes-1-9.adoc

```diff
@@ -246,7 +253,10 @@ If you do not add any configuration data, you can use the default data in the AP
 
 * With this update, support for `tekton.dev/v1alpha1` API pipeline resources, such as Pipeline, PipelineRun, Task, Clustertask, and TaskRun has been removed.	
 
-* With this update, the `tkn-pac setup` command has been removed. You can now re-use the functionality provided by the `tkn-pac setup` command by using the `tkn-pac webhook add` and `tkn-pac webhook update-token` commands. You can use the `tkn-pac webhook add` command to re-add a webhook to an existing Git repository. You can use the `tkn-pac webhook update-token` command to update the personal provider access token for an existing Secret object in the Git repository.	
+* With this update, the `tkn-pac setup` command has been removed. You can now re-use the functionality provided by the `tkn-pac setup` command by using the `tkn-pac webhook add` and `tkn-pac webhook update-token` commands. You can use the `tkn-pac webhook add` command to re-add a webhook to an existing Git repository. You can use the `tkn-pac webhook update-token` command to update the personal provider access token for an existing Secret object in the Git repository.
+
+* With this update, a namespace that runs a pipeline with default settings does not apply the `pod-security.kubernetes.io/enforce:privileged` label to a workload.
```

Comment:
@rupalibehera  @ppitonak - I have added the inputs as per your feedback. Kindly have a look and let me know your comments, if any.

Reply:
@vdemeester could you please check if this looks correct

Reply:
I think that is correct, not sure exactly how is the wording around this in the rest of the openshift docs, but it seems alright there.